lib/galaxy/workflow/modules.py

```diff
@@ -227,17 +233,18 @@ class SubWorkflowModule( WorkflowModule ):
     # - Second pass actually turn RuntimeInputs into inputs if possible.
     type = "subworkflow"
     name = "Subworkflow"
-    default_name = "Subworkflow"
 
     @classmethod
     def from_dict( Class, trans, d, **kwds ):
         module = Class( trans )
         if "subworkflow" in d:
-            module.subworkflow = d["subworkflow"]
+            module.subworkflow = d[ "subworkflow" ]
         elif "content_id" in d:
-            content_id = d["content_id"]
-            module.subworkflow = SubWorkflowModule.subworkflow_from_content_id( trans, content_id )
-        module.label = d.get( "label", None ) or None
+            from galaxy.managers.workflows import WorkflowsManager
```

Comment:
I guess this import is here because of the circular import in `lib/galaxy/managers/workflows.py`:
```
from galaxy.workflow.modules import module_factory, is_tool_module_type, ToolModule, WorkflowModuleInjector
```